### ISSUE TITLE
Can't find a route with mtb:name tag

### ISSUE THREAD

**Issue by HarelM** [IsraelHikingMap/Site#1768]
## What I expect to happen
To be able to find a way that has a mtb:name tag

## What really happened
It does not appear in the search results

## What I did that caused the issue - step by step
Search for יוסי 2

## My environment
* Operating System and version: any
* Web Browser and version: any

## Things I think the developers should know, images, links etc.
The way exists in the database but it has a very low factor (0.5) as it doesn't have any interesting tags.
It was also missed in the exact search.
So basically two bugs here...

**Comment by HarelM** [comment]
There are other issues when looking into this more closely:
1. These routes do not have a `name` tag so the left panel doesn't know how to present their name
2. These routes do not have a proper language associated with them, so if we want to include them in the bicycle routes layer we'd need to make sure they have a language so that they will appear according to the selected language (this should be fairly simple to fix though)

Bottom line, we need to have a lot of logic that was done to the `name` tag behave the same for the `mtb:name` tag and that's a lot of work, which I'm not sure has a good ROI.

The current UX will also cause issues in the following scenario:
1. search for `יוסי 2`
2. click edit
3. edit the name 
4. save

This will add `name` value to OSM which is not what we want...

**Comment by HarelM** [comment]
I've added some safeguard around editing a title, I hope this will help in this case.
Also added `poiMerged` attribute to merged features to prevent editing.
Here are some image of how the ben shemen forest will look after this will go live:
![image](https://user-images.githubusercontent.com/3269297/196703337-4e774d07-9b01-4f6f-b37a-ee9958560c2c.png)
Zooming out will cluster those as usual:
![image](https://user-images.githubusercontent.com/3269297/196703454-c4f49db8-e218-48c2-beef-2fe08c26f0db.png)

cc: @vallleyofdawn @Tsurha

**Comment by HarelM** [comment]
This was a lot more simple than I expected. I needed to add the logic for `mtb:name` only in a handful of places.
Overall I think this is a good addition to the bike-routes-POI-layer and to the site and app.
I assume people will want to show off their bikes and vigor by adding images and description to those routes. :-)
